sycl/test/abi/layout_accessors.cpp

```diff
@@ -1,6 +1,8 @@
 // RUN: %clangxx -fsycl -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s | FileCheck %s
 // REQUIRES: linux
```

Comment:
is there any reason to not run it on windows?

Reply:
Windows version only differs in class names (due to different standard library implementation)